### PR TITLE
Stealthex new app

### PR DESCRIPTION
**Pull request by DazkStealth**
## Description
- Adding Stealthex app with its widget.
## How Has This Been Tested?
- Checked widget operation with a test swap
## Types of changes
Adding new app that includes https://www.npmjs.com/package/@stealthex-io/widget

### CODE REVIEW

**Comment by IAmKio** [comment]
@DazkStealth do you mind pulling the latest staging into your project please as there are conflicts currently - thank you!

**Comment by DazkStealth** [comment]
@IAmKio sry haven`t noticed. Done it!

**Comment by IAmKio** [comment]
@DazkStealth could you please check the styling naming you're using - once we load your app and then go back to the home feed or open any of the Action Bar menus - the text styles / backgrounds etc have all been overridden. Please see the screenshot below:

Screenshot 2024-07-22 at 15 02 17

**Comment by IAmKio** [comment]
@DazkStealth thanks again for your updated submission - we just noticed that you've overridden the `html` style in the global.css file, could you please remove / comment it out and move that style to your application scope please?

https://github.com/pillarwallet/x/pull/90/commits/9cd2ea0f09969d9ca4c6b55b11ffd0e364c988ba#diff-d9cbf41d20b9faa34b9066b78856527aca74367f903adf9fc5b00d03bbab0191R34

**Comment by DazkStealth** [comment]
@IAmKio so i think it`s done.

File: src/apps/stealthex/lib/styles/resets.css
Comment (IAmKio) [inline src/apps/stealthex/lib/styles/resets.css:1]: Can we remove this reset please, our own global styles system should be responsible for this

File: src/apps/stealthex/lib/styles/simplex.css
Comment (IAmKio) [inline src/apps/stealthex/lib/styles/simplex.css:1]: I think this answers my question above

**Review by IAmKio — CHANGES_REQUESTED** [review]
We're nearly there just a few questions and suggestions - thank you!

**Comment by IAmKio** [comment]
@DazkStealth finally made it through the review - just a few questions, we're nearly there!

**Comment by DazkStealth** [comment]
@IAmKio Hey! I`ve done what you requested. Apologies for the delay.

**Comment by IAmKio** [comment]
@DazkStealth could you please check this error we got whilst running the code checks in this branch:

```[eslint] 
src/apps/stealthex/components/exchange/swap/input/index.tsx
  Line 74:8:  React Hook useEffect has a missing dependency: 'pressable'. Either include it or remove the dependency array  react-hooks/exhaustive-deps```

Thanks!

**Comment by DazkStealth** [comment]
@IAmKio it's done!

**Comment by IAmKio** [comment]
> @IAmKio it's done!

Re-running checks...

**Review by IAmKio — APPROVED** [review]
LGTM - thanks for being our first app within the PillarX ecosystem! LFG!!! 🚀 Our QA team will test this once it's merged - feel free to merge this when you're ready

**Comment by DazkStealth** [comment]
@IAmKio Looks like i do not have write access to merge it. Can you help me?

**Comment by IAmKio** [comment]
@DazkStealth done ✨

**Pull request merged**
